Tests/FunctionalTests/util/FunctionalTestBase.swift

```diff
@@ -263,7 +263,7 @@ class FunctionalTestBase: XCTestCase {
         }
 
         for expectedRequest in expectedNetworkRequests {
-            let waitResult = expectedRequest.value.await(timeout: 2)
+            let waitResult = expectedRequest.value.await(timeout: 10)
```

Comment:
Can we make the timeout configurable on this API if needed just for certain tests?

Reply:
We could make it configurable, but all network requests in Edge are going to need an increased timeout due to the async nature of the hit queue.